src/data/mockData.ts

```diff
@@ -0,0 +1,144 @@
+import { Schedule } from '../models'
+
+export const mockData: Schedule = {
+	classes: {
+		'class-1': {
+			id: 'class-1',
+			title: 'CS1'
+		},
+		'class-2': {
+			id: 'class-2',
+			title: 'CS2'
+		},
+		'class-3': {
+			id: 'class-3',
+			title: 'CS3'
+		},
+		'class-4': {
+			id: 'class-4',
+			title: 'CS4'
+		},
+		'class-5': {
+			id: 'class-5',
+			title: 'CS5'
+		},
+		'class-6': {
+			id: 'class-6',
+			title: 'CS6'
+		},
+		'class-7': {
+			id: 'class-7',
+			title: 'CS7'
+		},
+		'class-8': {
+			id: 'class-8',
+			title: 'CS8'
+		},
+		'class-9': {
+			id: 'class-9',
+			title: 'CS9'
+		},
+		'class-10': {
+			id: 'class-10',
+			title: 'CS10'
+		},
+		'class-11': {
+			id: 'class-11',
+			title: 'CS11'
+		},
+		'class-12': {
+			id: 'class-12',
+			title: 'CS12'
+		},
+		'class-13': {
+			id: 'class-13',
+			title: 'CS13'
+		},
+		'class-14': {
+			id: 'class-14',
+			title: 'CS14'
+		},
+		'class-15': {
+			id: 'class-15',
+			title: 'CS15'
+		},
+		'class-16': {
+			id: 'class-16',
+			title: 'CS16'
+		},
+		'class-17': {
+			id: 'class-17',
+			title: 'CS17'
+		},
+		'class-18': {
+			id: 'class-18',
+			title: 'CS18'
+		},
+		'class-19': {
+			id: 'class-19',
+			title: 'CS19'
+		},
+		'class-20': {
+			id: 'class-20',
+			title: 'CS20'
+		},
+		'class-21': {
+			id: 'class-21',
+			title: 'CS21'
+		},
+		'class-22': {
+			id: 'class-22',
+			title: 'CS22'
+		},
+		'class-23': {
+			id: 'class-23',
+			title: 'CS23'
+		},
+		'class-24': {
+			id: 'class-24',
+			title: 'CS24'
+		}
+	},
+	semesters: {
+		'semester-1': {
+			id: 'semester-1',
+			title: 'Fall',
+			classIds: ['class-1', 'class-2', 'class-3']
+		},
+		'semester-2': {
+			id: 'semester-2',
+			title: 'Spring',
+			classIds: ['class-4', 'class-5']
+		},
+		'semester-3': {
+			id: 'semester-3',
+			title: 'Summer 1',
+			classIds: ['class-6']
+		},
+		'semester-4': {
+			id: 'semester-4',
+			title: 'Summer 2',
+			classIds: ['class-7']
+		},
+		'semester-5': {
+			id: 'semester-5',
+			title: 'Fall',
+			classIds: ['class-8', 'class-9', 'class-10']
+		},
+		'semester-6': {
+			id: 'semester-6',
+			title: 'Spring',
+			classIds: ['class-11', 'class-12']
+		},
+		'semester-7': {
+			id: 'semester-7',
+			title: 'Summer 1',
+			classIds: ['class-13']
+		},
+		'semester-8': {
+			id: 'semester-8',
+			title: 'Summer 2',
+			classIds: ['class-14']
+		}
+	}
+}
```

Comment:
I think the data definition for Schedule that we made on Monday is a little different, but if this is sample data that's fine.